src/renderer/login/components/LoginButton/LoginButton.scss

```diff
@@ -0,0 +1,13 @@
+.actions {
```

Comment:
Can we rename this to `.loginButton` to match the component name?  This is what we've done in all our other component.